src/main/java/uk/gov/digital/ho/hocs/info/api/UserService.java

```diff
@@ -32,15 +32,13 @@ public UserService(KeycloakService keycloakService, CaseworkClient caseworkClien
         this.stageTypeService = stageTypeService;
     }
 
-    @Cacheable("users")
     public List<UserDto> getAllUsers() {
         log.info("Retrieving all users from Keycloak");
         List<UserDto> users = keycloakService.getAllUsers().stream().map(user -> UserDto.from(user)).collect(Collectors.toList());
         log.info("Found {} users", users.size());
         return users;
     }
 
-    @CachePut("users")
     public List<UserDto> refreshUserCache() {
```

Comment:
This can be removed - no longer called? If it is, it shouldn't be as we don't have any caching now...